docs/contracts.rst

```diff
@@ -491,6 +491,218 @@ and the arguments are ambiguous.
+
+.. code-block::
+
+    >>> #  pragma solidity >=0.4.22 <0.6.0;
+    ...
+    ... #   contract ArraysContract {
+    ... #      bytes2[] public bytes2Value;
+
+    ... #      constructor(bytes2[] memory _bytes2Value) public {
+    ... #          bytes2Value = _bytes2Value;
+    ... #      }
+
+    ... #      function setBytes2Value(bytes2[] memory _bytes2Value) public {
+    ... #          bytes2Value = _bytes2Value;
+    ... #      }
+
+    ... #      function getBytes2Value() public view returns (bytes2[] memory) {
+    ... #          return bytes2Value;
+    ... #      }
+    ... #  }
+
+.. doctest::
+
+    >>> abi = '''
+    ...  [
```

Comment:
This is a pretty large block to be in the docs.  You might hide this inside of a [`.. testsetup::`](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testsetup) directive so that it doesn't render for the user.  Then you can do what you did below for the bytecode:

```
>>> # abi = ...
```